ucp/_libs/tests/test_server_client.py

```diff
@@ -0,0 +1,94 @@
+import multiprocessing as mp
+import os
+from queue import Empty as QueueIsEmpty
+
+import pytest
+
+from ucp._libs import ucx_api
+from ucp._libs.arr import Array
+from ucp._libs.utils_test import blocking_recv, blocking_send
+
+mp = mp.get_context("spawn")
+
+
+def test_listener_ip_port():
+    ctx = ucx_api.UCXContext()
+    worker = ucx_api.UCXWorker(ctx)
+
+    def _listener_handler(conn_request):
+        pass
+
+    listener = ucx_api.UCXListener(worker=worker, port=0, cb_func=_listener_handler)
+
+    assert isinstance(listener.ip, str) and listener.ip
+    assert (
+        isinstance(listener.port, int) and listener.port >= 0 and listener.port <= 65535
+    )
```

Comment:
I think leaving this test in its own file as it was originally makes more sense, after all this doesn't semantically match a server/client test as this file name suggests.

Reply:
Fixed